tools/travis-publish-github-comment.sh

```diff
@@ -109,11 +112,18 @@ if [ -z "$AWS_SECRET_ACCESS_KEY" ]; then
     REPORTS_URL_BODY="Reports are not uploaded"$'\n'
     remove_ct_log_links
 else
-    rewrite_log_links_to_s3
     REPORTS_URL="$(reports_url)"
     CT_RUN_URL="$(ct_run_url)"
     SMALL_CT_URL="$(ct_small_url)"
-    REPORTS_URL_BODY="Reports [root](${REPORTS_URL}) / [big]($CT_RUN_URL) / [small]($SMALL_CT_URL)"$'\n'
+    REPORTS_SMALL_URL_BODY=""
+    if [ -d big_tests/ct_report ]; then
+        rewrite_log_links_to_s3
+        REPORTS_BIG_URL_BODY="/ [big]($CT_RUN_URL)"
```

Comment:
Will the `REPORTS_BIG_URL_BODY` var be undefined if big tests are not run? I see `REPORTS_SMALL_URL_BODY` is set to `""` before the `if`, I think the same could be done for `REPORTS_BIG_URL_BODY`